### DISCUSSION TITLE
Keep enderchest addictional rows data

### DISCUSSION THREAD

**Discussion by the1mason** [PurpurMC/Purpur#1120] [__Dismissed__] [answered]
A player that has additional rows filled with items in their enderchest loses their items after enderchest permissions are downgraded.
For example:
A player with the member tier is downgraded to the player tier, and their additional row is disappeared (as expected), but after getting their tier back they get an empty row with all their previous items disappearing.
I think that it could be better if we will keep additional rows hidden (not deleted) if a player doesn't have view permissions (maybe as a separate config option).

**Comment by BillyGalbreath** [comment, marked as answer]
Duplicate #907
